google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/InsertAllRequestTest.java

```diff
@@ -215,14 +212,25 @@ public void testEquals() {
     compareInsertAllRequest(INSERT_ALL_REQUEST11, INSERT_ALL_REQUEST11);
   }
 
+  @Test(expected=UnsupportedOperationException.class)
+  public void testImmutable() {
+    InsertAllRequest.RowToInsert row = InsertAllRequest.RowToInsert.of(new HashMap<String,Object>());
+    row.getContent().put("zip", "zap");
+  }
+
+  @Test
+  public void testNullOK() {
+    InsertAllRequest.RowToInsert.of(Collections.singletonMap("foo", null));
+  }
```

Comment:
This tests that nulls are ok, but I'm not seeing a test that verifies that the inserted row is null (or at least that whatever is stored or passed along is null).